toc_subprojects/project-reviews-subproject/governance-review-template.md

```diff
@@ -1,222 +1,208 @@
-* CNCF Code of Conduct is cross-linked from other governance documents.
+**Document adoption of the CNCF Code of Conduct**
+<br>
+**Incubating:** Required | **Graduated:** Required
 
-<!--- The CNCF CoC should be present in all appropriate places in the project, which 
-includes their Community or main repository, and public website at a minimum.  Other
-community and governance documentation should refer to the CoC; for example, role
-removal process should mention that people can be removed for CoC violations.  
-If the project is handling their own CoC reports rather than just sending them 
-directly to the CNCF CoCC, you need to check if this is adequately documented, including
-who receives the reports and how and where they're evaluated. --->
 
-### Subprojects
+**CNCF Code of Conduct is cross-linked from other governance documents.**
+<br>
+**Incubating:** Required | **Graduated:** Required
 
-* All subprojects, if any, are listed.
-* If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.
 
-<!--- If the project has subprojects, plugins, or other divisions define them here. For each, is ownership and operation of clearly described? Are any standing committees/teams fully described, including listing their members? Does it conform to, align, and is it within scope of the governance expectations of the project?  Does is list communications channels and standing meetings?
-A subproject audit is often very useful to projects because this part of things tends to 
-bit-rot quite a bit, and you can help them see where they need to update it. 
-If the project has a full listing of suboprojects use the text below.  Completeness hinges on whether subproject listings include full ownership, communications, and alignment info. --->
 
-The project maintains a [listing of subprojects](LINK).  This listing appears to be [Very Complete|Mostly Complete|Incomplete].
+### Subprojects
+
+**All subprojects, if any, are listed.**
+<br>
+**Incubating:** Required | **Graduated:** Required
 
-<!--- if the project does not maintain a listing, or if the listing is incomplete, then
-fill in the audit info below.  Otherwise, delete it.  Ownership and Operation is "who runs this subproject", Communications includes meetings, channels and email, and Project Alignment is "what part of the project is this, and how mature is it".  --->
 
-The project includes the following sub-projects, plugins, and other notable divisions:
+**If the project has subprojects: subproject leadership, contribution, maturity
+status documented, including add/remove process.**
+<br>
+**Incubating:** Suggested | **Graduated:** Required
+
 
-| Area | Ownership and Operation | Communications | Project Alignment | Notes |
-|:-----|:-----------------------:|:---------------:|:------------------|:---|
-|*sub-project*| Complete/Partial/Missing | Complete/Partial/Other | Complete/Partial/Conflict | |
```

Comment:
Why was this removed? This table allows for the efficient formatting of this information and understanding at a glance each sub-project's governance within the overarching project.

Reply:
I'll put it back with a link to the table generator, mostly people have put some pretty hard to read entries in there.